lib/handleRequest.js

```diff
@@ -13,8 +13,10 @@ const inputSchemaError = fastJsonStringify(schemas.inputSchemaError)
 function handleRequest (req, res, params, context) {
   var method = req.method
   var headers = req.headers
-  var request = new context.Request(params, req, urlUtil.parse(req.url, true).query, headers, req.log)
-  var reply = new context.Reply(res, context, request)
+  var Request = context.Request
+  var Reply = context.Reply
+  var request = new Request(params, req, urlUtil.parse(req.url, true).query, headers, req.log)
+  var reply = new Reply(res, context, request)
```

Comment:
Does the old code prevent V8 from optimizing the constructor call?

Reply:
not really, but I had to fix a bad bug in my change and I forgot these in. I can revert this bit if you want.